gcp/kubernetes/templates/run_batch.jsonnet

```diff
@@ -36,7 +36,7 @@ function(job_name, command, project_dir, uid, fsgroup,
         },
         {
           name: "PYTORCH_PRETRAINED_BERT_CACHE",
```

Comment:
Is the actual environment variable still called `PYTORCH_PRETRAINED_BERT_CACHE`? If so, can we leave it as `bert_cache` in our code as well?

Reply:
Yeah—they stuck with the legacy name in pytorch_transformers. Assuming it doesn't make a difference, I'd still rather use the new name internally, to make it clear that it's relevant to more than just BERT.